recipes/pkg-config/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+
+libtoolize
+aclocal
+autoreconf -i
```

Comment:
Nothing for you to do, but I think our staged-recipes image is different to our feedstock's Linux image, and the latter doesn't have autoreconf (right @jjhelmus?). We are in the process of fixing that, but it may mean your feedstock fails initially.